apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -441,52 +435,47 @@ class ReviewTab extends Component {
           {javalabMsg.noCodeReviewUntilStudentEditsCode()}
         </div>
       );
-    } else {
-      if (!initialLoadCompleted) {
-        return (
-          <div style={styles.loadingContainer}>
-            <Spinner size="large" />
-          </div>
-        );
-      }
+    }
 
+    if (!initialLoadCompleted) {
       return (
-        <div style={styles.reviewsContainer}>
-          <div style={styles.reviewHeader}>
-            {this.renderBackToMyProject(this.onClickBackToProject)}
-            {this.renderPeerDropdown(reviewablePeers, this.onSelectPeer)}
-            {this.renderReadyForReviewCheckbox()}
-          </div>
-          {errorSavingReviewableProject && (
-            <div style={styles.peerReviewErrorMessage}>
-              {javalabMsg.togglePeerReviewError()}
-            </div>
-          )}
-          <div style={styles.commentsSection}>
-            <div style={styles.messageText}>
-              {viewAsCodeReviewer || viewAs === ViewType.Teacher
-                ? javalabMsg.feedbackBeginningPeer({
-                    peerName: projectOwnerName
-                  })
-                : javalabMsg.feedbackBeginning()}
-            </div>
-            {this.renderComments(
-              comments,
-              !isReadyForReview && viewAs !== ViewType.Teacher
-            )}
-            {this.renderCommentEditor(forceRecreateEditorKey)}
-          </div>
+        <div style={styles.loadingContainer}>
+          <Spinner size="large" />
         </div>
       );
     }
+
+    return (
+      <div style={styles.reviewsContainer}>
+        <div style={styles.reviewHeader}>
```

Comment:
Nit: do we need this div if header is empty? Could we put it in renderHeader() if not?

Reply:
My follow-up PR (in progress) moves things around a bit more. I ended up getting rid of renderHeader. So I'll leave this as-is for now as it gets cleaned up in the next one:
https://github.com/code-dot-org/code-dot-org/pull/42716/files